mock-server/app/http.php

```diff
@@ -317,12 +316,11 @@
     ->param('x', '', new Text(100), 'Sample string param')
     ->param('y', '', new Integer(true), 'Sample numeric param')
     ->param('z', null, new ArrayList(new Text(256), APP_LIMIT_ARRAY_PARAMS_SIZE), 'Sample array param')
-    ->param('file', [], new File(), 'Sample file param', skipValidation: true)
+    ->param('payload', [], new File(), 'Sample file param', skipValidation: true)
```

Comment:
this change is not required right? we can keep the param name as file itself?

Reply:
This a quick test to determine that we aren't using hardcoded checks against the `file` param name